packages/helm-charts/testnet/templates/_helpers.tpl

```diff
@@ -199,7 +199,6 @@ spec:
           --metrics \
           --mine \
           --networkid=${NETWORK_ID} \
-          --nodekey=/root/.celo/account/{{ .Node.name}}PrivateKey \
```

Comment:
It looks like in this setup we would still have the private key at /root/.celo/account/*PrivateKey. Where is this file created?

Reply:
At least here the code is dead & is removed in the proxy infra PR that will be merged soon